Packs/FeedGithub/Integrations/FeedGithub/FeedGitHub.yml

```diff
@@ -0,0 +1,172 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Github Feed
+  version: -1
+configuration:
+- defaultvalue: 'true'
+  display: Fetch indicators
+  name: feed
+  required: false
+  type: 8
+- defaultvalue: https://api.github.com
+  display: Server URL
+  name: url
+  type: 0
+  additionalinfo: The REST API URL
+  section: Connect
+  required: false
+- defaultvalue: ""
+  display: Owner
+  name: owner
+  type: 0
+  additionalinfo: Username of the repository owner
+  section: Connect
+  required: true
+- defaultvalue: ""
+  display: Repository / Path to fetch
+  name: repo
+  type: 0
+  additionalinfo: The repository name
+  section: Connect
+  required: true
+- defaultvalue: ""
+  display: Feed type
+  name: feedType
+  options:
+  - YARA
+  - STIX
+  - IOCs
+  type: 15
+  additionalinfo: Predefined list of indicator types
+  section: Connect
+  required: true
+- display: Head branch name
+  name: branch_head
+  type: 0
+  required: true
+  defaultvalue: main
+  additionalinfo: 'The name of the main branch to which you would like to compare'
+- display: files extensions To Fetch
+  name: extensions_to_fetch
+  type: 16
+  required: true
+  defaultvalue: txt, yar, json
+  options:
```

Comment:
these are not the only options right? custom values can be given

Reply:
Indeed, there is a choice from the list
And you can add free text